app/models/concerns/translatable_attachment.rb

```diff
@@ -20,4 +20,12 @@ def t_attached(name, locale = I18n.default_locale)
     end
     send("#{name}_#{locale}")
   end
+
+  def t_attached_or_default(name, locale = I18n.locale)
+    if locale != I18n.default_locale && t_attached(name, locale).present?
+      t_attached(name, locale)
+    elsif t_attached(name, I18n.default_locale).present?
+      t_attached(name, I18n.default_locale)
+    end
```

Comment:
I'm likely reading what this method is doing totally wrong so please do ignore this if i am but i'm wondering if this could be simplified (I'm guessing `t_attached` returns nil if not found?)
```
  def t_attached_or_default(name, locale = I18n.locale)
    t_attached(name, locale) || t_attached(name, I18n.default_locale) || nil
  end
```
?